great_expectations/core/usage_statistics/schemas.py

```diff
@@ -906,6 +906,17 @@
     ],
 }
 
+cloud_migrate_schema = {
+    "$schema": SCHEMA,
+    "title": "cloud-migrate",
+    "type": "object",
+    "properties": {
+        "organization_id": {"type": "string", "maxLength": 256},
+    },
+    "required": ["organization_id"],
+    "additionalProperties": False,
+}
+
```

Comment:
Are we creating these jsonschema's by hand? 👀

Reply:
Sort of, we use python to create the actual json schema for the validator. I'm sure there is a better way :)

Reply:
I'm excited for that future :) But yeah a bit out of scope to convert right now. Looking forward to it!